model/registry/deprecated/http.yaml

```
@@ -7,37 +7,37 @@ groups:
       - id: method
         type: string
         brief: 'Deprecated, use `http.request.method` instead.'
-        stability: deprecated
+        deprecated: "Renamed to `http.request.method`."
         examples: ["GET", "POST", "HEAD"]
       - id: status_code
         type: int
         brief: 'Deprecated, use `http.response.status_code` instead.'
-        stability: deprecated
+        deprecated: "Renamed to `http.response.status_code`."
         examples: [200]
       - id: scheme
         type: string
         brief: 'Deprecated, use `url.scheme` instead.'
-        stability: deprecated
+        deprecated: "Renamed to `url.scheme` instead."
         examples: ['http', 'https']
       - id: url
         type: string
         brief: 'Deprecated, use `url.full` instead.'
-        stability: deprecated
+        deprecated: "Renamed to `url.full`."
         examples: ['https://www.foo.bar/search?q=OpenTelemetry#SemConv']
       - id: target
         type: string
         brief: 'Deprecated, use `url.path` and `url.query` instead.'
-        stability: deprecated
+        deprecated: "Split to `url.path` and `url.query."
         examples: ['/search?q=OpenTelemetry#SemConv']
       - id: request_content_length
         type: int
         brief: 'Deprecated, use `http.request.header.content-length` instead.'
-        stability: deprecated
+        deprecated: "Renamed to `http.request.header.content-length`."
```

(I'm also good with current text both here and below calling it a rename)
```suggestion
        deprecated: "Replaced by `http.request.header.content-length`."
```

---

Thanks, I like `replaced by` better